### ISSUE TITLE
模型在验证集上计算时，出现logits都是负数的情况，会是什么问题引起的呢？

### ISSUE THREAD

**Issue by Chandler-top** [gaohongkui/GlobalPointer_pytorch#24]
请问作者，目前使用该模型，对其他数据集进行实验时，发现训练集得到的 logits 有正数，但在做完一轮训练，开始在验证集上进行实验时，logits 得到的都是负数。即：metric_utils.py文件 下的

    def get_evaluate_fpr(self, y_pred, y_true):
        y_pred = y_pred.cpu().numpy()

   查看 y_pred ，最大值仅为 -1.xxx，而最小值是-inf。因此造成，R = 0， Y=0， 无法计算 Recall

 不知道 ，是什么原因造成 验证集 上的模型会生成 logits 为负数的情况，即没有预测出来。

**Comment by gaohongkui** [comment]
可以尝试更换预训练模型
或者调整超参数，例如学习率、优化器、scheduler 等
如果都没有效果的话，也可能是代码有问题，你可以试试苏剑林原版的实现 https://github.com/bojone/GlobalPointer 是否也有这个问题。如果原版实验效果好的话，欢迎反馈
